crates/subspace-networking/src/behavior/custom_record_store.rs

```diff
@@ -260,35 +262,39 @@ impl<'a> RecordStorage<'a> for NoRecordStorage {
     }
 }
 
-#[derive(Clone, Debug, Serialize, Deserialize)]
+use parity_scale_codec::{Decode, Encode};
+
+#[derive(Clone, Debug, Serialize, Deserialize, Decode, Encode)]
```

Comment:
If we don't use serde and will never be (this is a private type) we shouldn't derive it either.